### DISCUSSION TITLE
How to drop the two fields: "_airbyte_ab_id" and "_airbyte_emitted_at" from the output dump?

### DISCUSSION THREAD

**Discussion by hirads** [airbytehq/airbyte#34526] [Connector Questions]
Hi, 

I'm setting up a connection from Redshift to S3 but couldn't find any way to drop the two fields "_airbyte_ab_id" and "_airbyte_emitted_at" from the output. Anyone has an idea how to ignore them in the connection? 

ps. I've read [here](https://docs.airbyte.com/using-airbyte/core-concepts/basic-normalization#normalization-metadata-columns) that they are keeping track of important information, but I expected this to happen somewhere in the background and not on the final data dump.

**Comment by sajarin** [comment]
The fields "_airbyte_ab_id" and "_airbyte_emitted_at" are system fields added by Airbyte for data synchronization and tracking purposes. Currently, there is no built-in functionality to exclude these fields from the output.

However, you can create a transformation step after the extraction to remove these fields. This would involve creating a new table without these fields and copying the data over.

**Comment by meyerovb** [comment]
It would be faster (and maybe easier) to use unload or switch to spectrum depending on your use case

[a comment by an automated account is not shown]
